docs/modules/ROOT/pages/commands.adoc

```diff
@@ -56,6 +56,11 @@ The following commands can be used to format the text in the doc comments:
 |
 |===
 
+== Full List of Commands
```

Comment:
We don't need that (the nav is more visible already). The one that already exists also needs to be removed.